src/test/java/org/retest/rebazer/service/BitbucketServiceTest.java

```diff
@@ -68,16 +68,15 @@ public void rebaseNeeded_should_return_true_if_headOfBranch_isnt_equal_to_lastCo
 
 	@Test
 	public void isApproved_should_return_false_if_approved_is_false() {
-		final PullRequest pullRequest = mock( PullRequest.class );
 		final String json = "{participants: [{\"approved\": false}]}\"";
+		when( pullRequest.getUrl() ).thenReturn( "" );
```

Comment:
You use Mockito's `when` on an object that is not a mock. What was wrong with `mock( PullRequest.class )`? Did it break some tests?

Reply:
This was a leftover.